desktop/core/src/desktop/settings.py

```diff
@@ -42,6 +42,8 @@
 from aws.conf import is_enabled as is_s3_enabled
 from azure.conf import is_abfs_enabled
 
+reload(sys)
+sys.setdefaultencoding('utf-8')
```

Comment:
Thanks!

Just this is hardcoding, did you update https://github.com/cloudera/hue/blob/master/desktop/conf.dist/hue.ini#L153 too and why not use the value there instead?